service/dap/server.go

```diff
@@ -724,6 +739,7 @@ func (s *Server) onSetExceptionBreakpointsRequest(request *dap.SetExceptionBreak
 
 func (s *Server) onConfigurationDoneRequest(request *dap.ConfigurationDoneRequest) {
 	if s.args.stopOnEntry {
+		//TODO: execute a blocking state call here
```

Comment:
Is this to check if it is already running or stopped? Why blocking? I think the idea would be to halt if it is running.

Reply:
Sure, that's also fine.